docs/source/install/systemrequirements.rst

```diff
@@ -0,0 +1,28 @@
+System Requirements
+===================
+
+|st2| requires 64-bit version of Ubuntu/Debian, RHEL, or CentOS. The table below lists the supported
+versions, along with Vagrant Boxes Vagrant Boxes and Amazon AWS instances we use for
+testing. Yes, using exactly the same boxes will improve your experience.
```

Comment:
You removed `.. include:: __64bit_note.rst` and didn't put it here?

Is this intentional? I still think it would be good to leave the note here - people still some how miss only 64 bit is supported so more prominent note on that helps...